tests/tests.py

```diff
@@ -479,75 +377,3 @@ def test_unquote_s3(self, getenv, os):
 @pytest.mark.parametrize("was, now", modify_filename_data)
 def test_modify_targz_filename(was, now):
     assert ingester.modify_filename(was, addition="_") == now
-
-
-class TestPublicationLogic:
```

Comment:
This test class feels relevant still

Reply:
this PR is not ready for review, sorry I should edit it. I have anoother simpler one just for the publish change